packages/@react-aria/utils/src/useScrollWheel.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2021 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {RefObject, useEffect} from 'react';
+
+// scroll wheel needs to be added not passively so it's cancelable, small helper hook to remember that
+export function useScrollWheel({onScroll, capture}: {onScroll: (e) => void, capture: boolean}, ref: RefObject<HTMLElement>) {
```

Comment:
Should this go in `@react-aria/interactions` rather than utils? We could also automatically handle preventing default in here maybe (and probably not expose the raw DOM event like the other interaction hooks).